src/Controller/ListPageRssController.php

```diff
@@ -164,12 +192,57 @@ public function build(NodeInterface $node): CacheableResponse {
    *
    * @param \Drupal\node\NodeInterface $node
    *   The node containing the list.
+   * @param \Drupal\Core\Cache\CacheableMetadata $cache_metadata
+   *   The current cache metadata.
    *
    * @return array
    *   Array of items to be rendered.
    */
-  protected function getItemList(NodeInterface $node): array {
-    return [];
+  protected function getItemList(NodeInterface $node, CacheableMetadata $cache_metadata): array {
+    $configuration = ListPageConfiguration::fromEntity($node);
```

Comment:
We are missing the sorting: `Always sort by updated date, DESC. The RSS needs to show latest items.`

At the moment, we are sorting based on the bundle third party setting (see `executeList()`). So we need to make a ListPageConfiguration option where we can pass a different sort and use that if it's there instead of the content type.

Reply:
Good job. The test also confirms it.